src/web/components/Caption.tsx

```diff
@@ -93,30 +118,29 @@ export const Caption = ({
         );
     };
 
+    if (!captionText) return null;
+
     return (
-        <figure className={figureStyle}>
-            {children}
-            {captionText && (
-                <>
-                    <figcaption
-                        className={cx(
-                            captionStyle,
-                            shouldLimitWidth && limitedWidth,
-                            { [captionPadding]: padCaption },
-                        )}
-                    >
-                        <span
-                            className={cx(
-                                iconStyle,
-                                display === 'immersive' && hideIconBelowLeftCol,
-                            )}
-                        >
-                            <TriangleIcon />
-                        </span>
-                        {getCaptionHtml()} {displayCredit && credit}
-                    </figcaption>
-                </>
+        <figcaption
+            className={cx(
+                captionStyle,
+                shouldLimitWidth && limitedWidth,
+                !isOverlayed && bottomMargin,
+                isOverlayed && overlayedStyles,
+                {
+                    [captionPadding]: padCaption,
+                },
             )}
-        </figure>
+        >
+            <span
+                className={cx(
+                    iconStyle,
+                    display === 'immersive' && hideIconBelowLeftCol,
+                )}
+            >
+                <TriangleIcon />
+            </span>
+            {getCaptionHtml()} {displayCredit && credit}
```

Comment:
I think `getCaptionHtml()` should be move into the rendering, having it as its own function doesn't make much sense IMO

Reply:
Agreed. It's not that big and only used once. I'll do that refactor now 👍